monai/networks/losses/dice.py

```diff
@@ -49,26 +49,25 @@ def forward(self, pred, ground, smooth=1e-5):
 
         psum = pred.float()
         if self.do_sigmoid:
```

Comment:
Since do_sigmoid is False by default this changes the default of behaviour of this loss to apply no activation. This will break the examples which use DiceLoss.

Reply:
yes, the integration test was updated earlier: https://github.com/Project-MONAI/MONAI/blob/master/tests/integration_unet2d.py#L42

Reply:
The examples in the examples folder which use Dice need to be similarly updated..

Reply:
the current example doesn't work since we upgraded to use ignite 0.3.0...
how about a separate PR after merging https://github.com/Project-MONAI/MONAI/pull/56 dedicated to fixing the examples? I'll take note of this loss function issue.